be/src/util/mem_info.cpp

```diff
@@ -97,6 +98,8 @@ void MemInfo::process_cache_gc(int64_t& freed_mem) {
     freed_mem +=
             StoragePageCache::instance()->get_page_cache_mem_consumption(segment_v2::DATA_PAGE);
     StoragePageCache::instance()->prune(segment_v2::DATA_PAGE);
+    // TODO add freed_mem
+    SegmentLoader::instance()->prune();
```

Comment:
Why do we need to call `prune()` here?

Reply:
with the pr: https://github.com/apache/doris/pull/17011, the `prune()` here seems not necessary.

Reply:
minor gc, the remaining memory is less than 10% of the total memory, so prune outdated segment cache

Reply:
not conflict with periodic gc in #17011